qunicorn_core/api/provider_api/root.py

```diff
@@ -44,4 +44,4 @@ class RootView(MethodView):
     @PROVIDER_API.response(HTTPStatus.OK, RootSchema())
     def get(self):
         """Get the urls of the next endpoints of the provider api to call."""
-        return RootData(root=url_for("provider-api.ServicesView", _external=True))
+        return RootData(root=url_for("provider-api.ProviderView", provider_id=1, _external=True))  # provider_id=1 only a dummy value
```

Comment:
Wie funktioniert das mit der provider_id in Root, an welcher Stelle wird die relevant? Und wieso brauchen wir sie?